dih_models/parameters.py

```
@@ -9094,6 +9108,212 @@ def _pd_compound_fv_abolition(C, g, r, N):
+    latex_symbol=r"Cost_{RTT,life}",
+    hide_ci=True,
+)
+
+STATE_RTT_FDA_BUDGET_EQUIVALENT_HOURS = Parameter(
+    float(STATE_RTT_PHILANTHROPIC_COST_TOTAL) / float(FDA_ANNUAL_PROGRAM_BUDGET) * HOURS_PER_YEAR,
+    manual_ref="knowledge/appendix/state-right-to-trial-impact.qmd",
+    source_type="calculated",
+    description="Hours of the FDA annual program budget equal to the full central philanthropic launch cost for universal state Right to Trial. This is a scale comparison, not a claim about FDA cost-effectiveness.",
+    display_name="Universal State Right to Trial Cost in FDA Budget Hours",
+    unit="hours",
+    formula="STATE_RTT_PHILANTHROPIC_COST_TOTAL ÷ FDA_ANNUAL_PROGRAM_BUDGET × 8,760",
```

Ruff (0.16.1)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)